app/assets/javascripts/modules/sticky-element-container.js

```diff
@@ -10,77 +10,77 @@
 (function (Modules, root) {
   'use strict'
 
-  var $ = root.$
-  var $window = $(root)
-
   Modules.StickyElementContainer = function () {
     var self = this
 
-    self._getWindowDimensions = function _getWindowDimensions () {
+    self.getWindowDimensions = function () {
       return {
-        height: $window.height(),
-        width: $window.width()
+        height: root.innerHeight,
+        width: root.innerWidth
       }
     }
 
-    self._getWindowPositions = function _getWindowPositions () {
+    self.getWindowPositions = function () {
       return {
-        scrollTop: $window.scrollTop()
+        scrollTop: root.scrollY
       }
     }
 
-    self.start = function start ($el) {
-      var $element = $el.find('[data-sticky-element]')
-      var _hasResized = true
-      var _hasScrolled = true
-      var _interval = 50
-      var _windowVerticalPosition = 1
-      var _startPosition, _stopPosition
+    self.start = function ($el) {
```

Comment:
The sticky functionality no longer appears to work in IE10 and IE11 (probably something to do with how those older browsers calculate height). I don't think fixing that should be a blocking issue for this PR but it would be worth noting this somewhere so that we can address it separately.

Reply:
😱 oh no!  There appears to be a lot of complexities in calculating the height of elements. Again, I'll raise separate PR to address this.

Reply:
Raised [this card](https://trello.com/c/W73JrGKm/315-fix-sticky-element-in-government-frontend-not-working-in-ie-10-11) to address this issue.